libRALFit/test/example_module.f90

```diff
@@ -3004,11 +3004,7 @@ subroutine error_message_tests(options,fails)
 
      fails = 0
      n = 2
-     m = 67
-
-     allocate(params%x_values(m))
```

Comment:
Bug fix, double allocation of these arrays. `generate_data_example` already allocates them. Fixes 1st memleak